artifacts/deploy/200_rbac.yaml

```diff
@@ -40,7 +40,7 @@ rules:
 
   # to have the power to reconcile request(s)
   - apiGroups:
-    - operator.apps.openshift.io
+    - operator.openshift.io
```

Comment:
I guess this apigroup does not have to be the same with https://github.com/openshift/run-once-duration-override/blob/127caf32039d4bcecccdc14fc8484c4667c01763/pkg/api/api.go#L4

Reply:
nope. The one references is for the admission api. This one is for the operator. It's confusing.